.github/workflows/ci.yml

```diff
@@ -41,7 +41,7 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Install rust nightly toolchain
-        uses: dtolnay/rust-toolchain@nightly
+        uses: dtolnay/rust-toolchain@nightly-2023-07-03
```

Comment:
```suggestion
        uses: dtolnay/rust-toolchain@master
        with:
          toolchain: nightly-2023-07-03
```

Reply:
The README of the action should be updated :stuck_out_tongue:

Reply:
Ah no. My brain should be updated :sweat_smile: